SVG/SVG Use Cases/css/svgUseCases.css

```diff
@@ -0,0 +1,101 @@
+.featuredIcons svg {
+	min-height: 60px;
+	margin: auto;
+}
+
+
+.priceContainer img {
+	display: block;

```

Comment:
Write display:inline-block and remove margin:auto as wrapper have text-align:center